### PR TITLE
[cloud_infra_center] add storage template when deploying nodes

### PR DESCRIPTION
**Pull request by shhull**
This PR contains below changes:
1) add `volume_type_id` when creating virtual machine with boot volume
2) change all tasks name

Verified three situations on zvm test stand:
1) disk_type=scsi and volume_type_id=XXXXX
1.1) add new compute node
2) disk_type=scsi
2.1) add new compute node
3) disk_type=dasd

### CODE REVIEW

File: z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-pre-check/tasks/main.yaml [hunk 1]
```diff
@@ -87,4 +87,28 @@
 - name: Check server group quota
   fail:
     msg: "Server group number is not enough, current: {{ server_group.stdout_lines[0] }}, required: 1"
-  failed_when: "server_group.stdout_lines[0]|int < 1"
\ No newline at end of file
+  failed_when: "server_group.stdout_lines[0]|int < 1"
+
+- name: Get storage provider
+  shell: openstack volume service list -f csv | grep cinder-volume |grep -E 'enabled|up' |head -n1|awk -F , '{print$4$5}' |tr -d '"'
```
Comment (dongyanyang) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-pre-check/tasks/main.yaml:93]: this cmd is not checking the storage provider of the specified volume_type_id.
as long as there is one cinder-volume service up and running, the check will always be true. ()

suggest:
1. get the volume_backend_name from  "capabilities:volume_backend_name='v7k60'" in output of `openstack volume type show $$volume_type_id`
2. check this specific cinder-volume host status from output of 'openstack volume service list'

File: z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-pre-check/tasks/main.yaml [hunk 2]
```diff
@@ -87,4 +87,28 @@
...
+- name: Get storage template
+  shell: openstack volume type list -c ID -f value | grep {{ volume_type_id }} | wc -l
+  register: storage_template
+  when:  
+  - volume_type_id is defined
+  - disk_type == "scsi"
+
+- name: Check if storage provider is configured
+  fail:
+    msg: "Storage provider is not enabled and active."
```
Comment (dongyanyang) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-pre-check/tasks/main.yaml:108]: either not enabled or not active

File: z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-pre-check/tasks/main.yaml (hunk 1 above)
Comment (jichenjc) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-pre-check/tasks/main.yaml:93]: is this done at line 111-114? 
@dongyanyang

Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-pre-check/tasks/main.yaml:93]: capabilities:volume_backend_name='v7k60' was file `cinder-<storagehostname>.conf` content, I can not get this file through ansible host, not any ICIC nodes.

I think this task was used to check the cinder-volume service, I will remove the status check for storage provider.

And for the volume_type_id we have another task to check it.
```
- name: Get storage template
  shell: openstack volume type list -c ID -f value | grep {{ volume_type_id }} | wc -l
  register: storage_template
  when:  
  - volume_type_id is defined
  - disk_type == "scsi"
```

**Pull request merged**